recipes/magics-python/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "magics-python" %}
+{% set pypi_name = "Magics" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: b852ddc368116f9a3a8d9e778e7d82eb2f086601c5ff3cc61e2ed2928812d498
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - numpy
+    - python
+
+test:
+  # magics-python fails to import without the magics C++ library, but magics is currently built
```

Comment:
You can fix this by adding a file called `yum_requirements.txt` to the recipe directory containing:
```
mesa-libGL
mesa-dri-drivers
libselinux
libXdamage
libXxf86vm
```
This will cause conda-forge to install OpenGL on the host before building the recipe.

Reply:
Missed this in the docs - looks neat. Thanks, I'll give it a go :)

Reply:
Looks like it's worked nicely